lib/generators/component/templates/stimulus_view.html.slim.erb

```diff
@@ -0,0 +1,6 @@
+div class="<%= component_class_name %>"
```

Comment:
@andrzejsliwa Missing the `data-controller`

Reply:
done